test/blackbox-tests/test-cases/foreign-stubs/fs-mode-dependant/fsmd-exe.t/stubs_same_exe.ml

```diff
@@ -0,0 +1,3 @@
+external stub_byte_and_native : unit -> int = "caml_b_and_n"
+
+let () = Printf.printf "Byte (0) and native (0) ? %i\n" (stub_byte_and_native ())
```

Comment:
string is incorrect

Reply:
Not sure it is, in this test both byte and native should return 0.

Reply:
I don't understand the test, what should we understand if the test prints 0?

Reply:
Well since this stub is not mode-dependent the test should print 0 (which is an arbitrarily chosen value) in both case. There is some comment in the `run.t` file:
```

But stubs_same_exe does not have mode dependent stubs and should
always output 0
  $ dune exec ./stubs_same_exe.exe
  Byte (0) and native (0) ? 0

  $ dune exec ./stubs_same_exe.bc.exe
  Byte (0) and native (0) ? 0
  ```